db/seeds.rb

```diff
@@ -5,5 +5,49 @@
 #
 #   cities = City.create([{ name: 'Chicago' }, { name: 'Copenhagen' }])
 #   Mayor.create(name: 'Emanuel', city: cities.first)
-role = Role.create!([{name: 'organizer'}])
-user = User.create!([{name: 'Wolverine', roles: role, github_handle: 'Adminis28'}])
\ No newline at end of file
+
+
+# Temp commented out seeds from upstream repo
+  # role = Role.create!([{name: 'organizer'}])
+  # user = User.create!([{name: 'Wolverine', roles: role, github_handle: 'Adminis28'}])
+
+require 'FFaker' #TODO: do I need this?
+
+# JobOffer.create!( title: "Example Job Title",
+#                   company_name: "Example Company",
+#                   contact_name: "Example Name",
+#                   description: "Interesting details",
+#                   duration: "12 months",
+#                   location: "US",
+#                   misc_info: "tralala",
+#                   paid: true,
+#                   url: "http://www.google.com",
+#                   activated: true,
+#                   activated_at: Time.zone.now)
+
+    4.times do
+       title = FFaker::Job.title
+       company_name = FFaker::Company.name
+       contact_name = FFaker::Name.name
+       description = FFaker::HipsterIpsum.phrase
+       duration = "#{Random.new.rand(3..24)} months"  #type is string
+       location = FFaker::Address.country             #type is string
+       misc_info = FFaker::Company.bs                 #type is string
+       paid = FFaker::Boolean.random                  #type is bool
+       url = FFaker::Internet.http_url                #url is string
+       activated = true
+       activated_at = Time.zone.now
```

Comment:
There's nothing to be gained with these temporary variables. You could simply assign the values directly.

You should only really need these if you reference them more than once.